extra/CMakeLists.txt

```diff
@@ -1,63 +1,79 @@
+# Copyright (C) 2016-2019 Blue Brain Project
+#
+# See top-level LICENSE file for details.
+# =============================================================================
 
-get_directory_property(CORENRN_COMPILE_FLAGS COMPILE_DEFINITIONS)
+# =============================================================================
+# Prepare nrnivmodl-core script with correct compiler flags
+# =============================================================================
 
-if (CORENRN_COMPILE_FLAGS)
-    set(CORENRN_COMPILE_DEFS "-D${CORENRN_COMPILE_FLAGS}")
-    string(REPLACE ";" " -D" CORENRN_COMPILE_DEFS "${CORENRN_COMPILE_DEFS}")
+# extract the COMPILE_DEFINITIONS property from the directory.
+get_directory_property(CORENRN_COMPILE_FLAGS COMPILE_DEFINITIONS)
+if(CORENRN_COMPILE_FLAGS)
+  set(CORENRN_COMPILE_DEFS "-D${CORENRN_COMPILE_FLAGS}")
+  string(REPLACE ";"
+                 " -D"
+                 CORENRN_COMPILE_DEFS
+                 "${CORENRN_COMPILE_DEFS}")
 endif()
 
-# Link libraries
-# CMake does some magic to transform sys libs to -l<libname>. We replicate it
+# replicate CMake magic to transform system libs to -l<libname>
 foreach(link_lib ${CORENRN_LINK_LIBS})
-    get_filename_component(pth ${link_lib} DIRECTORY)
-    if(NOT pth)
-       string(APPEND CORENRN_LINK_DEFS " -l${link_lib}")
-    elseif("${pth}" MATCHES "^(/lib|/lib64|/usr/lib|/usr/lib64)$")
-        get_filename_component(libname ${link_lib} NAME_WE)
-        string(REGEX REPLACE "^lib" "" libname ${libname})
-        string(APPEND CORENRN_LINK_DEFS " -l${libname}")
-    else()
-        string(APPEND CORENRN_LINK_DEFS " ${link_lib}")
-    endif()
+  get_filename_component(path ${link_lib} DIRECTORY)
+  if(NOT path)
+    string(APPEND CORENRN_LINK_DEFS " -l${link_lib}")
+  elseif("${path}" MATCHES "^(/lib|/lib64|/usr/lib|/usr/lib64)$")
+    get_filename_component(libname ${link_lib} NAME_WE)
+    string(REGEX
+           REPLACE "^lib"
+                   ""
+                   libname
+                   ${libname})
+    string(APPEND CORENRN_LINK_DEFS " -l${libname}")
+  else()
+    string(APPEND CORENRN_LINK_DEFS " ${link_lib}")
+  endif()
 endforeach()
-#message("CORENRN_LINK_LIBS: ${CORENRN_LINK_LIBS}, CORENRN_LINK_DEFS: ${CORENRN_LINK_DEFS}")
 
-# Additional mods
-string(REPLACE ";" " " CORENRN_OPTMODS "${CORENRN_OPTMECH_MODS}")
-string(REPLACE ";" " " CORENRN_OPTMODS_OUTPUTS "${CORENRN_OPTMECH_OUTPUTS}")
-
-# PGI add --c++11;-A option for c++11 flag
-string(REPLACE ";" " " CXX11_STANDARD_COMPILE_OPTION "${CMAKE_CXX11_STANDARD_COMPILE_OPTION}")
+# PGI compiler add --c++11;-A option for c++11 flag
+string(REPLACE ";"
+               " "
+               CXX11_STANDARD_COMPILE_OPTION
+               "${CMAKE_CXX11_STANDARD_COMPILE_OPTION}")
 
 # MPI path might be multiple directories
-if (MPI_C_INCLUDE_PATH)
-    string(REPLACE ";" " -I" CORENRN_MPI_INCLUDES "${MPI_C_INCLUDE_PATH}")
+if(MPI_C_INCLUDE_PATH)
+  string(REPLACE ";"
```

Comment:
Don't you need to here, similarly to above also prepend a `-I` in front of the first path?

```
set(MPI_C_INCLUDE_ATH "-I${MPI_C_INCLUDE_PATH}")
```

Reply:
yes...I noticed that this is not used anywhere so I will remove it